clusterloader2/testing/load/configmap.yaml

```diff
@@ -6,7 +6,36 @@ metadata:
 immutable: true
 {{end}}
 data:
+{{if $ENABLE_DNSTESTS}}
```

Comment:
You don't have access to any variable that is not listed explicitly in templateFillMap in config.yaml